src/frontend/components/TranscriptSentence/index.spec.tsx

```diff
@@ -24,7 +24,18 @@ describe('<TranscriptSentence />', () => {
       <TranscriptSentence cue={cue} active={false} />,
     );
 
-    fireEvent.click(getByText('Lorem ipsum dolor sit amet.'));
+    const sentence = getByText('Lorem ipsum dolor sit amet.');
+
+    expect(sentence).not.toHaveStyleRule(
+      'background-color',
+      'rgba(242,94,35,0.25)',
```

Comment:
I think some kind of regex might be better for these tests? This would make them less likely to break on refactors while still ensuring the styling happens (eg. not test the exact color for example)

I think `jest-styled-components` supports this

Reply:
for the `not` test the second paramter can also be `undefined` if wnato to know if the style is missing.

Reply:
done in 796ba9e067c140b822f57e2ec1637932e1600083

Reply:
I also meant it for the case where there are style rules, with a regex we could make sure there's a background color or a solid border without enforcing the exact color in the test

WDYT?

Reply:
Just expection a string then. I made it 4c9b00d1f791a3f47725112f4486618a9472a2ca